### ISSUE TITLE
Adding relays not working until restart

### ISSUE THREAD

**Issue by jamesob** [mikedilger/gossip#499]
I've added a few relays and nothing is happening:

![image](https://github.com/mikedilger/gossip/assets/73197/cdfda708-3388-46c1-9110-fede714e6ba8)

![image](https://github.com/mikedilger/gossip/assets/73197/c22334be-0d9c-475c-bfb6-1f2c6afeaa5b)
![image](https://github.com/mikedilger/gossip/assets/73197/f2ad98a8-7850-4231-9781-212e6523c4ee)

After restarting, the relays appear to work properly.

**Comment by mikedilger** [comment]
Are you saying that things don't work until you restart, and that is an issue?  Or are you saying they still dont work?

**Comment by jamesob** [comment]
No data was retrieved from relays I added/configured until I restarted Gossip. But even now, I have no kind of stream or timeline showing.
![image](https://github.com/mikedilger/gossip/assets/73197/27e2f157-5019-4918-8ea7-4c42b689b110)

I do see a few entries in My Notes, but definitely not my npub's complete history.

I'm confused about what to do from here.

**Comment by mikedilger** [comment]
Navigate to Help > Help to get advise on how to setup gossip.

I'm working on a setup wizard that will make this smoother.

**Comment by jamesob** [comment]
Yeah, I read the help page - I'm still not understanding why a timeline isn't populating with all the npubs I've previously followed (on other apps). Maybe I'm missing something, but when I open e.g. Amethyst, I get a full timeline full of notes. In gossip I don't see anything, despite having added multiple relays. This is a very confusing user experience.

**Comment by jamesob** [comment]
Ah - I see, the key was pulling down People metadata. I'm a little surprised this doesn't happen automatically upon connecting some threshold of relays. That might be nice.

**Comment by mikedilger** [comment]
Yeah this will be fixed in the 'wizard'. Sorry. I know the client seems quite advanced, but we still have these really annoying problems that I (for example) never see since I've been setup for so long.

**Comment by dtonon** [comment]
Duplicate of https://github.com/mikedilger/gossip/issues/449

**Comment by dtonon** [comment]
@mikedilger I moved this issue from #477 and then reopened this issue.
